server/main.lua

```diff
@@ -39,12 +39,35 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleowner = GetVehicleOwner(plate)
+		local bolo, title, boloid = GetBoloStatus(plate, title, boloid)
+		local warrant, owner, incidentid = GetWarrantStatus(plate, owner, incidentid)

```

Comment:
Same comment here, the args `owner` and `incidentid` that you are passing to GetWarrantStatus are not defined. Looks like you dont need to pass these either, just `plate` should be enough since you are actually getting owner and incidentId as results from this function

Reply:
Amended code in new commit d13ea57